scripts/qa_crowbarsetup.sh

```diff
@@ -2341,6 +2341,16 @@ function addfloatingip()
     nova add-floating-ip "$instanceid" "$floatingip"
 }
 
+# by setting --dns-nameserver for subnet, docker instance gets this as
+# DNS info (otherwise it would use /etc/resolv.conf from its host)
+function adapt_dns_for_docker()
+{
+    # DNS server is the first IP from the allocation pool, or the
+    # second one from the network range
+    local dns_server=`neutron subnet-show fixed | grep allocation_pools | cut -d '"' -f4`
+    neutron subnet-update --dns-nameserver "$dns_server" fixed
```

Comment:
:+1:  in general - but the check and fail is still missing, please open a new PR